content/applications/finance/fiscal_localizations/italy.rst

```diff
@@ -49,387 +45,502 @@ section, click :guilabel:`Update info`. From here, fill out the fields:
+   and therefore, they require :guilabel:`Exoneration` kind and :guilabel:`Law Reference` filled in.
+
+.. image:: italy/tax-exemption.png
+   :alt: Tax Exemption Settings
+
+.. seealso::
+   There are many :guilabel:`Tax Exemption Kind (Natura)` and :guilabel:`Law Reference` codes.
+   Make sure you check the latest version available to get the latest information on:
+
+   - `Italian authorities documentation <https://www.agenziaentrate.gov.it/portale/web/guest/aree-tematiche/fatturazione-elettronica>`_
+   - `Official guide on Tax exemption  <https://www.agenziaentrate.gov.it/portale/documents/20143/451259/Guida_compilazione-FE-Esterometro-V_1.9_2024-03-05.pdf/67fe4c2d-1174-e8de-f1ee-cea77b7f5203>`_
 
 .. note::
-   You can :ref:`enable electronic invoicing for other sales and purchase journals
-   <e-invoicing/configuration>` than the default ones.
+   If you need to use a different kind of :guilabel:`Exoneration`, go to :menuselection:`Accounting
+   --> Configuration --> Taxes`, select a similar tax, then click on the cog icon and select
+   :guilabel:`Duplicate`. In the :guilabel:`Advanced Options` tab, add the :guilabel:`Exoneration`
+   and :guilabel:`Law Reference`. To confirm click on :guilabel:`Save`.
 
-You can check the current status of an invoice by the :guilabel:`Electronic invoicing` field. The
-XML file can be found in the **chatter** of the invoice.
+.. tip::
+   **Rename** your taxes in the :guilabel:`Name` field according to their :guilabel:`Exoneration` to
+   differentiate them easily.
 
-.. image:: italy/italy-test.png
-   :align: center
-   :alt: Electronic invoicing status (waiting for confirmation)
+.. _italy/reverse-charge:
 
-.. seealso::
-   :doc:`../accounting/customer_invoices/electronic_invoicing`
+Reverse charge
+==============
+
+The **reverse charge** mechanism is a VAT rule that shifts the liability to pay VAT from the
+supplier to the customer. The customers pay the VAT *themselves* to the :abbr:`AdE (Agenzia delle
+Entrate)` instead. There are different types:
+
+- | :guilabel:`Internal Reverse Charge` (for domestic sales)
+  | The VAT responsability is shifted to the buyer for certain categories of products and services
+    where the risk of tax evasion is higher.
+- | :guilabel:`External Reverse Charge` (for intra-EU sales)
+  | VAT is due in the country of the delivery or in the country where the service is performed.
+    When the buyer is itself an Italian business, then the EU offers a mechanism that allows the
+    seller to transfer his responsibility to the buyer.
+
+Invoices
+--------
+
+**Reverse-charged** customer invoices show no VAT amount, but the :abbr:`AdE (Agenzia delle
+Entrate)` requires the seller to specify the :guilabel:`Tax Exemption` reason and the :guilabel:`Law
+Reference` that enable the reverse-charge mechanism. Odoo provides a set of special 0% taxes that
+can be assigned to each reverse-charged invoice lines, representing the most
+commonly used configurations.
```

Comment:
This bit was a little messy, so I tried to follow TSB and JCS instructions.

Chiara didn't like the "tax evasion" sentence much, but I'm using it again because otherwise I don't know how to clarify Internal Reverse Charge.

Let me know what you people think, I'm open to suggestions.
Thank you all for the hard work. 🙏🏻